docs/_templates/layout.html

```diff
@@ -1,5 +1,15 @@
 {% extends "!layout.html" %}
 
+{% block header %}
+{%- if development %}
+<div id="development-warning" style="padding: .5em; text-align: center; background-color: #FFBABA; color: #6A0E0E;">
+    {% trans %}This document is for the development version.
+    For the stable version, see {% endtrans %}
+    <a href="/en/stable/{{ pagename }}{{ file_suffix }}">{% trans %}here{% endtrans %}</a>.
```

Comment:
Just a remark regarding the translation markers, not every language is able to put the link words at the end of the sentence. And the link itself contains a language marker and thus should really be part of the translation.

But since it's somewhat unlikely that the Cython documentation will get translated in the neat future, we can also leave it like this and fix it later (or never).